packages/oui-datagrid/README.md

```diff
@@ -0,0 +1,442 @@
+```
+
+### Responsive
+
+#### Basic
+
+```html:preview
+<div class="oui-datagrid-responsive-container">
+  <div class="oui-datagrid-responsive-container__sticky-container">
+    <div class="oui-datagrid-responsive-container__overflow-container">
+      <table class="oui-datagrid">
+        <thead class="oui-datagrid__headers">
+          <tr>
+            <th class="oui-datagrid__header oui-datagrid__header_sortable" tabindex="0">First Name</th>
+            <th class="oui-datagrid__header oui-datagrid__header_sortable-asc" tabindex="0">Last Name</th>
+            <th class="oui-datagrid__header oui-datagrid__header_sortable-desc" tabindex="0">Mother</th>
+            <th class="oui-datagrid__header oui-datagrid__header_sortable" tabindex="0">Father</th>
+            <th class="oui-datagrid__header oui-datagrid__header_sortable" tabindex="0">Email</th>
+            <th class="oui-datagrid__header oui-datagrid__header_sortable" tabindex="0">Phone</th>
+            <th class="oui-datagrid__header oui-datagrid__header_sortable" tabindex="0">Birth</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr class="oui-datagrid__row">
+            <td class="oui-datagrid__cell">Tammy</td>
+            <td class="oui-datagrid__cell">Adams</td>
+            <td class="oui-datagrid__cell">Howe, Clement</td>
+            <td class="oui-datagrid__cell">Mann, Ines</td>
+            <td class="oui-datagrid__cell">tadams27@1688.com</td>
+            <td class="oui-datagrid__cell">55-(821)413-9216</td>
+            <td class="oui-datagrid__cell">Aug 21, 1980</td>
+          </tr>
+          <tr class="oui-datagrid__row">
+            <td class="oui-datagrid__cell">Norma</td>
+            <td class="oui-datagrid__cell">Adams</td>
+            <td class="oui-datagrid__cell">Prohaska, Alysa</td>
+            <td class="oui-datagrid__cell">Rau, Raoul</td>
+            <td class="oui-datagrid__cell">nadams2n@histats.com</td>
+            <td class="oui-datagrid__cell">1-(512)242-1388</td>
+            <td class="oui-datagrid__cell">Sep 27, 1974</td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+  </div>
+</div>
+```
+
+#### Checkboxes and action-menus
+
+```html:preview
+<div class="oui-datagrid-responsive-container oui-datagrid-responsive-container_sticky-checkbox oui-datagrid-responsive-container_sticky-actions">
+  <div class="oui-datagrid-responsive-container__sticky-container oui-datagrid-responsive-container__sticky-container_scroll-begin
+  oui-datagrid-responsive-container__sticky-container_scroll-end">
+    <div class="oui-datagrid-responsive-container__overflow-container">
+      <table class="oui-datagrid">
+        <thead class="oui-datagrid__headers">
+          <tr>
+            <th class="oui-datagrid__header oui-datagrid__cell-sticky">
+              <div class="oui-checkbox">
+                <input class="oui-checkbox__input" id="oui-checkbox1-global" name="oui-checkbox1-global" type="checkbox" checked="checked">
```

Comment:
(optional) Whatch out your ids are not unique in page. This comment can be applied to other checkboxes in this page.

Reply:
fixed